src/quo/components/list_items/address/view.cljs

```diff
@@ -25,15 +25,16 @@
             [text/text
              {:size   :paragraph-1
               :weight :semi-bold
-              :style  {:color (colors/resolve-color network)}} (str (subs (name network) 0 3) ":")])
+              :style  {:color (colors/resolve-color network theme)}}
+             (str (subs (name network) 0 3) ":")])
```

Comment:
can we instead make network a shape similiar to what I have in this pr?
https://github.com/status-im/status-mobile/pull/17650
i.e 
```
{:network-name <keyword to resolve the color>
 :short-name <string to display the shorter name>}
```
```

Reply:
Yeah of course, will do